chart/values.yaml

```diff
@@ -43,6 +43,11 @@ securityContexts:
   pod: {}
   containers: {}
 
+# Default container level lifecycle hooks for every container
+# See:
+# https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
```

Comment:
I don't think we need to link folks to this everywhere. Easy enough to search for it.

```suggestion
```